cli/azd/ci-test.ps1

```diff
@@ -1,5 +1,5 @@
 param(
-    [string] $Timeout = '90m',
+    [string] $Timeout = '30m',
```

Comment:
Leaving this comment here for reverting if setting to 30m was for debugging purposes only.